payments-core/AndroidManifest.xml

```diff
@@ -32,7 +32,7 @@
         <activity
             android:name=".payments.StripeBrowserLauncherActivity"
             android:theme="@style/StripeTransparentTheme"
-            android:launchMode="singleTop"
+            android:launchMode="singleTask"
```

Comment:
Updated to singleTask

Reply:
Doesn't have to be here, but would be nice to add a comment as to why this is necessary.